tests/benchmarks/fixture-bootstrap.php

```
@@ -0,0 +1,447 @@
+			}
+
+			$stripe_settings = WC_Stripe_Helper::get_stripe_settings();
+			$required        = [ 'enabled', 'testmode', 'test_publishable_key', 'test_secret_key' ];
+			foreach ( $required as $key ) {
+				if ( empty( $stripe_settings[ $key ] ) ) {
+					throw new RuntimeException( sprintf( 'Benchmark fixture setup failed: Stripe setting "%s" is empty.', $key ) );
+				}
+			}
+
+			if ( 'yes' !== $stripe_settings['enabled'] || 'yes' !== $stripe_settings['testmode'] || 'yes' !== ( $stripe_settings['payment_request'] ?? '' ) ) {
+				throw new RuntimeException( 'Benchmark fixture setup failed: Stripe gateway and express checkout must be enabled in test mode.' );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '129', column '15'). (undefined)

(MissingImport)